functions-python/helpers/utils.py

```diff
@@ -40,6 +40,28 @@ def create_bucket(bucket_name):
         logging.info(f"Bucket {bucket_name} already exists.")
 
 
+def cors_configuration(bucket_name, origin=["*"], method=["GET"], max_age_seconds=3600):
```

Comment:
[question] how come we need a function to set the cors configuration when we can set it manually in GCP?